tests/e2e/EditProfile.Test.spec.js

```diff
@@ -16,9 +17,11 @@ module.exports = {
       .click('#edit-profile')
       .setValue('input[name="firstName"]', ' Editted')
       .setValue('input[name="lastName"]', ' Editted')
-      .assert.value('input[name="firstName"]', 'Hope Editted')
-      .assert.value('input[name="lastName"]', 'Ngerebara Editted')
+      .assert.value('input[name="firstName"]', 'admin2 Editted')
+      .assert.value('input[name="lastName"]', 'admin Editted')
       .click('button[id=edit]')
+      .waitForElementVisible('.toast', 3000)
+      .assert.containsText('.toast', 'Profile updated successfully')
```

Comment:
Fixed it here https://github.com/andela-hngerebara/Document-Management-System/pull/88